src/raftstore/coprocessor/split_check/mod.rs

```diff
@@ -60,11 +60,14 @@ impl Host {
         false
     }
 
-    pub fn split_key(mut self) -> Option<Vec<u8>> {
-        self.checkers
-            .drain(..)
-            .flat_map(|mut c| c.split_key())
-            .next()
+    pub fn split_keys(self) -> Vec<Vec<u8>> {
+        for mut checker in self.checkers {
+            let keys = checker.split_keys();
+            if !keys.is_empty() {
```

Comment:
Seems it always return empty or one key, why not use previous design `Option<xx>`?

Reply:
That's not true. The key count depends on the checker.

Reply:
Seems all checker would return empty or one key now?

Reply:
Yes. As the PR description says:

> make batch split possible.